content/drone-plugins/drone-anynines/index.md

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
Should this file be in a different PR?

Reply:
Good catch, I used the wrong base branch :D